tests/test_gndtruth.bats

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bats
+
+load setup_bats
+
+################################################################################
+# fn_cleanup                                                                   #
+################################################################################
+# ---------------------------------------------------------------------------- #
+#   Parameter 1: Extension of a groundtruth file which shall be deleted        #
+# Postcondition: /tmp/cloe_gndtruth.$1 deleted                                 #
+# Postcondition: /tmp/cloe_gndtruth.json deleted                               #
+# Postcondition: /tmp/cloe_gndtruth.msg.$1 deleted                             #
+################################################################################
+
+fn_cleanup() {
+    rm -f /tmp/cloe_gndtruth.json
+    rm -f /tmp/cloe_gndtruth.msg
+    [[ -z $1 ]] || rm -f /tmp/cloe_gndtruth.$1
+}
+
+################################################################################
```

Comment:
Please let's not use this banner-style documentation. It makes maintaining documentation a chore and it's also inconsistent with the rest of our coding style.

Reply:
Opinion:
1) A difference to the status-quo is in my eyes not relevant since an improvement is never too late. It is just the question whether we want to improve.
2) The cardinality for creating code/documentation is 1 for each time the code is touched. The cardinality for understanding code is *.

Reply:
True and true.

Unfortunately I personally find banner-comments very unreadable, and I find they don't play well with IDEs or editors that can fold code. I like navigating my functions and then expanding them to see the documentation. Or better, having my editor popup the documentation when I use the function.

I certainly like how thorough you are, although I think some of the comments are stating 1-to-1 what the code does, and this goes against rule no. 1 of most best-practice documents I read: https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/

I'd prefer to write code that's a little longer but then doesn't require as much commenting. For example, in bash functions, I find it's much more readable to do this:

```bash
my_func() {
  local suffix=$1
  local output_name=$2

  # ... use $suffix and $output_name
}
```

as opposed to just using `$1` and `$2` in the code and commenting what the first and second parameters are.